docs/website/docs/command-reference/init.md

```diff
@@ -14,7 +14,7 @@ The command can be executed in two flavors, either interactive or non-interactiv
 In interactive mode, you will be guided to choose:
 - a devfile from the list of devfiles present in the registry or registries referenced (using the `odo registry` command),
 - a starter project referenced by the selected devfile,
-- a name for the component present in the devfile.
+- a name for the component present in the devfile; this name should follow the [the Kubernetes naming convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names).
```

Comment:
Besides that, we have an additional requirement: the name must not be all-numeric. Maybe you could add that as well.
To me (and as I already reported so in #5652), "123" is a valid RFC 1123 value, accepted by Kubernetes as a resource name.

```suggestion
- a name for the component present in the devfile; this name must follow the [Kubernetes naming convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) and not be all-numeric.
```

Reply:
The suggested change seems to have disappeared..

Reply:
I think that's because I forgot to pull the remote changes and force pushed local changes. sorry about this.